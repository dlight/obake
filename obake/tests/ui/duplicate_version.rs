#[obake::versioned]
#[obake(version("0.1.0"))]
#[obake(version("0.1.0"))]
struct Foo {}

#[obake::versioned]
#[obake(version("0.1.0"))]
#[obake(version("0.1.0"))]
enum Bar {}

fn main() {}
