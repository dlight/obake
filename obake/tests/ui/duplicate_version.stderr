error: duplicate definition of version 0.1.0
 --> $DIR/duplicate_version.rs:3:17
  |
3 | #[obake(version("0.1.0"))]
  |                 ^^^^^^^

error: duplicate definition of version 0.1.0
 --> $DIR/duplicate_version.rs:8:17
  |
8 | #[obake(version("0.1.0"))]
  |                 ^^^^^^^
